server/views/pages/functionalSkills/macros/sortableTable.njk

```diff
@@ -0,0 +1,29 @@
+{#  -------------------------------------------------------------
+    Sortable table for the functional skills page
+    `config` argument is an object with the following properties:
+      * id - the id of the sortable table
+      * assessments - an array of assessment data, where each item is an instance of view model type Assessment
+    ------------------------------------------------------------- #}
+{% macro sortableTable(config) %}
+    <table class="govuk-table" data-module="moj-sortable-table" data-qa="{{ config.id }}-sortable-table">
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header app-u-width-20">Name</th>
```

Comment:
❤️ The fixed widths in the screenshot in the PR description look ace 👍 
Thank you